optimade/models/responses.py

```diff
@@ -67,17 +66,17 @@ class InfoResponse(Success):
 
 
 class EntryResponseOne(Success):
-    data: Union[EntryResource, Dict[str, Any], None] = Field(...)
-    included: Optional[Union[List[EntryResource], List[Dict[str, Any]]]] = Field(
+    data: Union[EntryResource, Dict[str, Any], None] = Field(...)  # type: ignore[assignment]
```

Comment:
If you add Dict[str, Any] and List[Dict[str, Any]] to the possible data types of the data field in Response class in jsonapi.py
the ignore messages can be removed from lines 69 and 76

Similarly if you replace 366 in jsonapi.py with `included: Optional[Union[List[Resource], List[Dict[str, Any]]]] = StrictField(`
you can remove the type ignores from lines 70 and 79

Reply:
I tried this, but as `ErrorResponse` also uses `Response` (which then breaks the OpenAPI schema again), I think its better to be strict at the top-level then excuse ourselves where necessary.